awx/main/models/inventory.py

```diff
@@ -910,7 +910,7 @@ def cleanup_task(cls, months_ago):
 
 class HostMetricSummaryMonthly(models.Model):
     """
-    HostMetric summaries computed by scheduled task <TODO> monthly
+    HostMetric summaries computed by scheduled task 'awx.main.tasks.system.host_metric_summary_monthly' monthly
```

Comment:
this module path `awx.main.tasks.system.host_metric_summary_monthly` looks like it's old, now that you put it into a new `handlers/` folder inside of the task folder. I don't love the idea of introducing a folder there, because what does "handlers" mean, and why is it applicable here? However, I do strongly support keeping separate modules, so like `awx/main/tasks/host_metric_summary_monthly_task.py` would make the most sense to me, personally.

Reply:
the module path is not old, it wasn't here at all, anyway it's part of the same feature/ADR, just splitted to two releases. 
Anyway I made handlers directory(module), because the rest of the files in `tasks` except "system.py" is quite unclear to me, what they actually represent. Handlers represent handlers for tasks, also usable for awx-manage commands. 
more questions in chat